docs(administration): add quota period accounting docs (#224)
reanahub/docs.reana.io: docs/advanced-usage/user-quotas/index.md

    means that the cluster administrators did not enable the user quotas feature on
    this REANA instance.

> ## Understand periodic CPU quota windows

Reviewer: 🤖 **Heading style inconsistent with sibling headings (LOW)**

Other H2 headings on this page are noun phrases ("Check quota limits and usage", "What to do when quotas are exceeded"). "Understand periodic CPU quota windows" is an imperative, which stands out.

Suggested fix — match the prevailing style:

```markdown
## Periodic CPU quota windows
```

or, if you'd like to keep the gerund hint:

```markdown
## Understanding periodic CPU quota windows
```

Author: Fixed

Some REANA deployments account CPU usage inside recurring quota periods, for
example every three months, instead of over the whole lifetime of your account.
Periodic accounting currently applies only to CPU quotas.

> When periodic CPU accounting is enabled, the profile page can show the current

Reviewer: 🤖 **"can show" hedges, and the section is missing the customary profile-page screenshot (LOW)**

Two small style issues for the same section:

1. "the profile page **can show** the current CPU accounting window" — hedging. Either it shows the window (when the feature is on) or it doesn't. "can" doesn't add information.

2. Every other major section on this page that refers to the UI includes a screenshot (`ui-profile-your-quota.png`, `ui-sort-by-most-disk-quota-used.png`, `ui-delete-workflow.png`, etc.). Since reana-ui PR #494 adds visible `startLabel` / `endLabel` text to the profile-page quota box, a screenshot here would match the page's convention and help users recognise the feature at a glance.

Suggested fix:

```markdown
When periodic CPU accounting is enabled, the profile page shows the current
CPU accounting window together with your CPU usage:

![CPU accounting window on the profile page](/images/ui-profile-cpu-quota-period.png)
```

Author: Fixed

    means that the cluster administrators did not enable the user quotas feature on
    this REANA instance.

> ## Periodic CPU quota windows

Reviewer: 🤖 **Section title hides what the feature does**

"Periodic CPU quota windows" describes the mechanism but not the user-visible benefit (the quota renews every N months based on cluster admin settings). Reframing around "renewable" makes the value clearer to end users.

```suggestion
## Renewable CPU quotas
```

Author: Fixed

Some REANA deployments account CPU usage inside recurring quota periods, for
example every three months, instead of over the whole lifetime of your account.

> Periodic accounting currently applies only to CPU quotas.

Reviewer: 🤖 **Awkward verb "to account" and inconsistent noun**

"Account CPU usage" reads like a translation; "track CPU usage" is more natural English. Also dropping "periodic accounting" in favour of the user-facing framing.

```suggestion
Some REANA deployments track CPU usage inside recurring quota periods, for
example every three months, instead of over the whole lifetime of your account.
This renewable-quota mechanism currently applies only to CPU quotas.
```

Author: Fixed

> Periodic accounting currently applies only to CPU quotas.

Reviewer: 🤖 **Add "As of REANA 0.95 release series" version notice (supersedes earlier body rewrite for this paragraph)**

Mirrors the `Since REANA 0.8.0` notice at the top of the page, which marks the quota feature as a whole — this new sub-feature deserves its own marker.

```suggestion
As of REANA 0.95 release series, some deployments track CPU usage inside
recurring quota periods, for example every three months, instead of over the
whole lifetime of your account. This renewable-quota mechanism currently
applies only to CPU quotas.
```

Author: Fixed

When periodic CPU accounting is enabled, the profile page shows the current
CPU accounting window together with your CPU usage. The start of each window is
derived from your account creation time and from the quota period length chosen

> by the administrators.

Reviewer: 🤖 **Standardize on "CPU quota period"**

Replaces "periodic CPU accounting", "CPU accounting window", and "quota period length" with the single noun "CPU quota period", matching the `quota_period_months` field name.

```suggestion
When renewable CPU quotas are enabled, the profile page shows the current
CPU quota period together with your CPU usage. The start of each period is
derived from your account creation time and from the period length chosen
by the administrators.
```

Author: Fixed

> !["CPU accounting window on the profile page"](/images/ui-profile-cpu-quota-period.png)

Reviewer: 🤖 **Alt text should match the chosen noun**

```suggestion
!["CPU quota period on the profile page"](/images/ui-profile-cpu-quota-period.png)
```

Author: Fixed

> !["CPU accounting window on the profile page"](/images/ui-profile-cpu-quota-period.png)

Reviewer: 🤖 **Apply the `screenshot-browser-mockup` CSS class (supersedes earlier alt-text suggestion for this line)**

Combines the alt-text rename from the first review with the macOS-chrome class.

```suggestion
!["CPU quota period on the profile page"](/images/ui-profile-cpu-quota-period.png){.screenshot-browser-mockup}
```

Reviewer: Alternatively, just crop the screenshot to show only the most important part, i.e. the quota period. It is not necessary to reproduce the whole browser window again. This will better attract eyes towards what's most important WRT quota periods.

<img width="615" height="414" alt="Screenshot 2026-05-18 at 15 02 43" src="https://github.com/user-attachments/assets/739af5b5-8ecc-4a3b-9d56-cffed3e0012d" />

BTW in the above image the limit is too generous, and the percentage is not very instructive (0%). It'll be better to retake the screenshot, set a limit to say 15 minutes, so that a consumption of about 2 minutes would create 13% of some such.

Author: Fixed, and updated the screenshot with a more informative limit!

If the administrators change your CPU quota period, the new period start is
stored immediately. However, the displayed CPU usage is recalculated only during
the next quota usage update cycle. Until then, the usage number may still

> reflect the previous accounting window.

Reviewer: 🤖 **"Previous accounting window" → "previous period"**

Keeps consistent terminology with the rest of the section.

```suggestion
If the administrators change your CPU quota period, the new period start is
stored immediately. However, the displayed CPU usage is recalculated only during
the next quota usage update cycle. Until then, the usage number may still
reflect the previous period.
```

Author: Fixed
